packages/modeling/src/utils/trigonometry.js

```diff
@@ -0,0 +1,25 @@
+
```

Comment:
Since this was added to user level utilities, then let’s expose the functions sin/cos in documentation.

Reply:
Added docs, types, and tests for these functions :+1: